packages/paste-website/src/pages/utilities/screen-reader-only/index.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: Screen Reader Only - Utilities
+description: The Screen Reader Only component should be used to visually hide content that may provide extra context to assistive technology users.
+---
+
+import {graphql} from 'gatsby';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {Text} from '@twilio-paste/text';
+import {Flex} from '@twilio-paste/flex';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Heading} from '@twilio-paste/heading';
+import {Card} from '@twilio-paste/card';
+import {Button} from '@twilio-paste/button';
+import {Anchor} from '@twilio-paste/anchor';
+import Changelog from '@twilio-paste/screen-reader-only/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Table, Thead, Tbody, Tr, Td, Th} from '../../../components/table';
+import {InlineCode} from '../../../components/Typography';
+
+export const pageQuery = graphql`
+  {
+    allPasteUtility(filter: {name: {eq: "@twilio-paste/screen-reader-only"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/screen-reader-only/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Screen Reader Only"
+  categoryRoute={SidebarCategoryRoutes.UTILITIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/utilities/screen-reader-only"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/utilities-screen-reader-only--default"
+  data={props.data.allPasteUtility.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About Screen Reader Only
+
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+When wrapped with the Screen Reader Only component, content is visually hidden but still available to assistive technology.
```

Comment:
I wonder if we should always refer to it as `ScreenReaderOnly` (one word) to disambiguate from some abstract "Screen reader only" thing.  Reason being, I had to read over this sentence twice to follow along.

Reply:
Does that work for headings though?